lib/geckodriver.js

```diff
@@ -0,0 +1,152 @@
+// transpile:main
+import events from 'events';
+import _ from 'lodash';
+import { logger } from 'appium-support';
+import { JWProxy } from 'appium-base-driver';
+import { SubProcess } from 'teen_process';
+
+const log = logger.getLogger('Geckodriver');
+
+const DEFAULT_HOST = '127.0.0.1';
+const DEFAULT_PORT = 29146;
+const SYSTEM_GECKODRIVER = '/usr/local/bin/geckodriver';
```

Comment:
we also need to support Windows :-P

Reply:
That is why it makes sense to look for the driver in PATH instead

Reply:
I added the geckodriverExecutable capability. Do we have another driver that would look for the driver in the system PATH?

Reply:
[here](https://github.com/appium/appium-adb/blob/8559abd0d96eff694bfd2aedf68e35c9a8915bda/lib/helpers.js#L90) you have an example